components/bl-dashlet-component/README.md

```diff
@@ -6,19 +6,20 @@ The component allows you to add dashlet to your UI-Builder application.
-| minHeight    | Number                            | 300           | Min Height Logic    | YES          | YES        | Allows to determine min-height                                                                                          |
-| maxHeight    | Number                            | 900           | Max Height Logic    | YES          | YES        | Allows to determine max-height                                                                                          |
+| Property     | Type                              | Default Value | Logic               | Data Binding | UI Setting | Description                                                                                                                                                           |
+|--------------|-----------------------------------|---------------|---------------------|--------------|------------|-----------------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| title        | Text                              |               | Title Logic         | YES          | YES        | Allows to determine the title for dashlet                                                                                                                             |
+| styleVariant | Select ["default", "alternative"] | "default"     |                     | NO           | YES        | Allows to determine the variant of style for dashlet                                                                                                                  |
+| height       | Number                            | 400           | Height Logic        | YES          | YES        | Allows to determine the height                                                                                                                                        |
+| width        | Number                            | 600           | Width Logic         | YES          | YES        | Allows to determine the width                                                                                                                                         |
+| localStorage | Checkbox                          | `false`       | Local Storage Logic | YES          | YES        | Allows to determine can save state(size, position, close or open) of dashlet in [Local Storage](https://developer.mozilla.org/en-US/docs/Web/API/Window/localStorage) |
+| resizing     | Checkbox                          | `false`       | Resizing Logic      | YES          | YES        | Allows to determine can resize or can't                                                                                                                               |
```

Comment:
Allows to determine whether resizing is allowed or not

Or something like that. The current phrase contains a mistake.

Same for other with 'can'